pkgs/top-level/all-packages.nix

```diff
@@ -20645,7 +20645,7 @@ in
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
   syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray { };
-  syncthingtray-minumal = libsForQt5.callPackage ../applications/misc/syncthingtray {
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray {
```

Comment:
when was this attribute added? If it's in 19.09 like this we need to backport a change.
And I guess setup an alias, which is unfortunate if anyone has that in their config.

Reply:
This was committed by me (`git blame` me :point_left::) in https://github.com/NixOS/nixpkgs/pull/65480.

Reply:
ah ok, no worries. If you make it a separate commit and add an alias to `aliases.nix` I can backport it for you.

Reply:
If you are talking about backporting the rename for me personally (as it seems I may be the only Nix user of this package) then I don't mind it at all - I can just change it in my own `nix-env`.

As for other users who may have come to rely on that, I think this change won't be too bad. We've already performed such renames, e.g https://github.com/NixOS/nixpkgs/pull/65724 .